zkevm-circuits/src/witness/rw.rs

```diff
@@ -865,6 +893,22 @@ impl From<&operation::OperationContainer> for RwMap {
                 })
                 .collect(),
         );
+        rws.insert(
+            RwTableTag::MemoryWord,
+            container
+                .memory_word
+                .iter()
+                .map(|op| Rw::MemoryWord {
+                    rw_counter: op.rwc().into(),
+                    is_write: op.rw().is_write(),
+                    call_id: op.op().call_id(),
+                    memory_address: u64::from_le_bytes(
+                        op.op().address().to_le_bytes()[..8].try_into().unwrap(),
+                    ),
```

Comment:
@silathdiir do you think this u64 limit is correct?

Reply:
Seems could be:
```
memory_address: op.op().address().0 as u64,
```

Since [struct MemoryAddress(pub usize)](https://github.com/scroll-tech/zkevm-circuits/blob/develop/eth-types/src/evm_types/memory.rs#L13C5-L13C37).